base/applications/sc/print.c

```diff
@@ -110,50 +36,63 @@ PrintServiceStatus(
 {
     _tprintf(_T("\tTYPE               : %x  "),
         (unsigned int)pStatus->dwServiceType);
+
     switch (pStatus->dwServiceType)
     {
         case SERVICE_KERNEL_DRIVER:
-            _tprintf(_T("KERNEL_DRIVER\n"));
+            _tprintf(_T("KERNEL_DRIVER"));
             break;
-
         case SERVICE_FILE_SYSTEM_DRIVER:
-            _tprintf(_T("FILE_SYSTEM_DRIVER\n"));
+            _tprintf(_T("FILE_SYSTEM_DRIVER"));
             break;
-
         case SERVICE_WIN32_OWN_PROCESS:
-            _tprintf(_T("WIN32_OWN_PROCESS\n"));
+            _tprintf(_T("WIN32_OWN_PROCESS"));
             break;
-
         case SERVICE_WIN32_SHARE_PROCESS:
-            _tprintf(_T("WIN32_SHARE_PROCESS\n"));
+            _tprintf(_T("WIN32_SHARE_PROCESS"));
             break;
-
         case SERVICE_WIN32_OWN_PROCESS + SERVICE_INTERACTIVE_PROCESS:
-            _tprintf(_T("WIN32_OWN_PROCESS (interactive)\n"));
+            _tprintf(_T("WIN32_OWN_PROCESS (interactive)"));
             break;
-
         case SERVICE_WIN32_SHARE_PROCESS + SERVICE_INTERACTIVE_PROCESS:
-            _tprintf(_T("WIN32_SHARE_PROCESS (interactive)\n"));
+            _tprintf(_T("WIN32_SHARE_PROCESS (interactive)"));
             break;
-
-        default : _tprintf(_T("\n")); break;
+        // default: // Nothing.
     }
 
+    _tprintf(_T("\n"));
```

Comment:
It is an unnesessary change as it adds an additional `_tprintf()` call and looks more ugly.

Reply:
Reverted.